build/docker/gaia-taskenv/Dockerfile

```diff
@@ -1,14 +1,7 @@
-from taskcluster/tester:0.3.6
+from taskcluster/tester:0.4.4
```

Comment:
Gotcha, sounds like we'll probably want to do something similar for gaia-taskenv instead of inheriting. Or, we could just roll the gaia-taskenv particularities into tester and just be done.